### PR TITLE
#467177: Documented inbox and discovery concepts

### PR DESCRIPTION
**Pull request by afuechsel**
Signed-off-by: Andre Fuechsel <a.fuechsel@telekom.de>

### CODE REVIEW

**Comment by maggu2810** [comment]
* I believe the linked resource "UPnP discovery service documentation" is missing
* IMHO there is no Bonjour protocol. Bonjour is the name of an implementation.

**Comment by maggu2810** [comment]
The link looks for the .html and not .md (ATM here on github).

**Comment by kaikreuzer** [comment]
> I believe the linked resource "UPnP discovery service documentation" is missing

No, this is correct as it will be made available on https://www.eclipse.org/smarthome/documentation/, where it is required that the links are .html. This new doc infrastructure needs to be documented itself though :-)

**Comment by kaikreuzer** [comment]
> IMHO there is no Bonjour protocol. Bonjour is the name of an implementation.

I agree, this should be mDNS instead.

File: docs/documentation/development/bindings/discovery-services.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+```java
+    @Override
+    protected synchronized void stopScan() {
+        super.stopScan();
+        removeOlderResults(getTimestampOfLastScan());
+    }
+```
+
+## UPnP Discovery
```
Comment (kaikreuzer) [inline docs/documentation/development/bindings/discovery-services.md]: It would have been nice to also have a section for the mDNSDiscoveryService here!

**Pull request merged**